src/axom/core/tests/CMakeLists.txt

```diff
@@ -89,6 +89,29 @@ if (ENABLE_MPI)
   endforeach()
 endif()
 
+#------------------------------------------------------------------------------
+# OpenMP GTests
+#------------------------------------------------------------------------------
+if (ENABLE_OPENMP)
+  set( core_openmp_tests
+       core_openmp_map.hpp
+       )
+
+  blt_add_executable( NAME       core_openmp_tests
+                      SOURCES    core_openmp_main.cpp
+                      HEADERS    ${core_mpi_tests}
+                      OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
+                      DEPENDS_ON ${utils_tests_depends} openmp
+                      FOLDER     axom/core/tests )
+
+  foreach( test_suite ${core_openmp_tests} )
+    get_filename_component( test_suite ${test_suite} NAME_WE )
+    axom_add_test( NAME          ${test_suite}_openmp
+                   COMMAND       core_openmp_tests --gtest_filter=${test_suite}*
+                   NUM_MPI_TASKS 1 )
```

Comment:
I think we have a way to specify the number of omp threads too

Reply:
`NUM_OMP_THREADS`

https://github.com/LLNL/axom/blob/57a6ef7ab50e113e4cf4b639657eb84ff10789c0/src/cmake/AxomMacros.cmake#L118-L127

See also: https://llnl-blt.readthedocs.io/en/develop/api/target.html#blt-add-test